plugins/node/opentelemetry-instrumentation-redis/test/redis.test.ts

```diff
@@ -47,7 +47,7 @@ const memoryExporter = new InMemorySpanExporter();
 
 const CONFIG = {
   host: process.env.OPENTELEMETRY_REDIS_HOST || 'localhost',
-  port: process.env.OPENTELEMETRY_REDIS_PORT || '63790',
+  port: process.env.OPENTELEMETRY_REDIS_PORT || 63790,
```

Comment:
Testing these changes proved difficult. The container listens on `63790`, while the default redis port is `6379`. The fix revolves around what the client and instrumentation do when dealing with default values. So if we provide an invalid port, the client will try (and fail) to connect, and the test will fail before getting to the instrumentation.

So I'm unsure what to do here.

Reply:
sounds not straightforward to test. I think the original intent of using non standard port was not to conflict with users environments that are already running redis.

If we try to send redis command to `6379`, will it end the span with an error, or simply not create any span? perhaps we can verify the recording of those default attributes on the erroneous span?

Reply:
That's a good idea, but this Redis instrumentation (unlike redis4) only creates spans for commands, and not for the connection. The connection happens when the client is created, before any commands are sent

If it's desirable I can add a span for making the connection - but that feels like it should be done for its own sake, and not just for testing for default values (though we can think of something like e.g. having it disabled by default, like a `createConnectionSpan` configuration value)